tests/e2e/specs/modules/analytics/setup-no-account-no-tag.test.js

```diff
@@ -88,9 +88,9 @@ describe( 'setting up the Analytics module with no existing account and no exist
 
 		await page.waitForSelector( '.googlesitekit-setup-module__inputs' );
 
-		await expect( page ).toMatchElement( '.googlesitekit-analytics__select-account .mdc-select__selected-text', { text: /select one.../i } );
-		await expect( page ).toMatchElement( '.googlesitekit-analytics__select-property .mdc-select__selected-text', { text: /select an account/i } );
-		await expect( page ).toMatchElement( '.googlesitekit-analytics__select-profile .mdc-select__selected-text', { text: /select an account/i } );
+		await expect( page ).toMatchElement( '.googlesitekit-analytics__select-account .mdc-select__selected-text', { text: '' } );
```

Comment:
This is because of how the `<Select>` works, right? Why did this behaviour change on us though?

Reply:
This is a decision that was made recently in the latest setup related work we did before this in Tag Manager.

Basically, instead of adding our own option for the (choose something) case, we don't and use the default behavior of the underlying component which is to position the floating label as a placeholder. Probably not the most accessible thing in the world, but this is what we decided on for now.

So it was an intentional change, to update the unselected state/choice.